presto-main/src/test/java/io/prestosql/sql/planner/TestLogicalPlanner.java

```diff
@@ -370,7 +392,9 @@ public void testLeftConvertedToInnerInequalityJoinNoEquiJoinConjuncts()
                 anyTree(
                         filter("O_ORDERKEY < L_ORDERKEY",
                                 join(INNER, ImmutableList.of(), Optional.empty(),
-                                        tableScan("orders", ImmutableMap.of("O_ORDERKEY", "orderkey")),
+                                        node(
```

Comment:
keep `node(FilterNode.class,` in same line, e.g:
```
node(FilterNode.class,
  tableScal("orders")
...
```
we decided to use this schema for plan matching as it's easier to read.